kernel/src/sys/syscall.c

```diff
@@ -138,7 +138,19 @@ static void syscall_wm(registers_t* regs) {
         case WM_CMD_EVENT: {
                 wm_param_event_t* param = (wm_param_event_t*) regs->ecx;
                 wm_get_event(param->win_id, param->event);
-        } break;
+            } break;
+        case WM_CMD_IS_DRAGGED: {
+            list_t* item = wm_get_window(regs->ecx);
+
+            if (item != NULL) {
+                wm_window_t* win = list_entry(item, wm_window_t);
+                return win->being_dragged;
+            } else {
+                printke("the given window id (%d) is unknown", regs->ecx);
+                regs->eax = false;
+                break;    
+            }
+            } break;
```

Comment:
Indentation is wrong + `return` in a void function. I also still don't like `being_dragged` being part of the wm_window state, but I can fix that after merging if needed. I can explain syscalls if you want, I know they are not the most straightforward concept.